src/clj/ataru/hakija/hakija_routes.clj

```diff
@@ -433,11 +433,14 @@
       :summary "Get cached koulutustyypit from koodisto"
       (let [codes (koodisto/get-koulutustyypit koodisto-cache)]
         (response/ok codes)))
-    (api/POST "/has-applied" []
+    (api/POST "/has-applied" [:as request]
       :summary "Check if a person has already applied"
       :body [has-applied-params ataru-schema/HasAppliedParams]
-      (let [{:keys [haku-oid ssn email eidas-id]} has-applied-params]
-        (log/info "Checking has-applied" has-applied-params)
+      (let [{:keys [haku-oid ssn email eidas-id]} has-applied-params
+            session (:session request)
+            oppija-session-from-db (some-> (get-in request [:cookies "oppija-session" :value])
+                                           (oss/read-session))]
+        (log/info "Checking has-applied" has-applied-params session oppija-session-from-db)
```

Comment:
Voisi harkita että lokittaisi session sisällöstä vain (:logged-in oppija-session-from-db)-booleanin. Ne muut tiedot kyllä löytyvät auditlokilta tarvittaessa

Reply:
Jos näin, niin ehkäpä tuo olisi tosiaan parempi ratkaisu - ei sitten ainakaan vahingossa päädy lokille mitään ylimääräistä.